packages/mui-base/src/useSwitch/useSwitch.ts

```diff
@@ -115,11 +155,25 @@ export default function useSwitch(props: UseSwitchParameters): UseSwitchReturnVa
     onBlur: createHandleBlur(otherProps),
```

Comment:
The event handlers are not needed here as the input is hidden and won't be directly interacted with by users.